### PR TITLE
HV-1651 Contribute additional language translations from OpenLiberty

### PR DESCRIPTION
**Pull request by aguibert**
The OpenLiberty application server uses Hibernate Validator and provides
translated messages in 15 languages, 6 of which are new languages not
currently supported by Hibernate Validator.  The new languages are:

 - Italian (it)
 - Japanese (ja)
 - Polish (pl)
 - Romanian (ro)
 - Chinese (Simplified) (zh)
 - Chinese (Traditional) (zh_TW)

Signed-off-by: Andrew Guibert <andy.guibert@gmail.com>

### CODE REVIEW

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties [hunk 1]
```diff
@@ -1,27 +1,56 @@
...
-javax.validation.constraints.NotBlank.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotEmpty.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotNull.message     = nesm\u00ed b\u00fdt null
-javax.validation.constraints.Null.message        = mus\u00ed b\u00fdt null
-javax.validation.constraints.Past.message        = mus\u00ed b\u00fdt v minulosti
-javax.validation.constraints.Pattern.message     = se mus\u00ed shodovat s "{regexp}"
-javax.validation.constraints.Size.message        = velikost mus\u00ed b\u00fdt mezi {min} a {max}
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+# NLS_MESSAGEFORMAT_NONE
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: Is this header really necessary? It looks like implementation specific?

It also says UNICODE whereas the files are not Unicode encoded.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties [hunk 2]
```diff
@@ -1,27 +1,56 @@
-javax.validation.constraints.AssertFalse.message = mus\u00ed b\u00fdt ne
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: AFAICS, the permissions of all the files were changed from 644 to 755. I don't think it's a good idea as these files shouldn't be executable.

**Comment by gsmet** [comment]
add to whitelist

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties (hunk 2 above)
Comment (aguibert) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: yea that was unintentional.  My mac must have done that when I copied the files across -- I'll correct the perms

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties (hunk 1 above)
Comment (aguibert) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: These are comments that our translation team's tooling adds.  I left them in so I can more easily diff between OpenLiberty and Hibernate Validator's copy of the message files.

**Comment by aguibert** [comment]
Review comments have been addressed, ready for re-review @gsmet

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 3]
```diff
@@ -1,40 +1,53 @@
...
-javax.validation.constraints.Null.message           = muss null sein
-javax.validation.constraints.Past.message           = muss in der Vergangenheit liegen
-javax.validation.constraints.PastOrPresent.message  = muss in der Gegenwart oder Vergangenheit liegen
-javax.validation.constraints.Pattern.message        = muss auf Ausdruck "{regexp}" passen
-javax.validation.constraints.Positive.message       = muss gr\u00F6\u00DFer 0 sein
-javax.validation.constraints.PositiveOrZero.message = muss gr\u00F6\u00DFer oder gleich 0 sein
-javax.validation.constraints.Size.message           = muss zwischen {min} und {max} liegen
+javax.validation.constraints.AssertFalse.message     = muss false sein.
+javax.validation.constraints.AssertTrue.message      = muss true sein.
+javax.validation.constraints.DecimalMax.message      = muss kleiner ${inclusive == true ? 'oder gleich ' : ''}{value} sein
```
Comment (gunnarmorling) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: The usage of the trailing dot is a bit inconsistent. I don't think we should add it.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties [hunk 4]
```diff
@@ -1,27 +1,53 @@
...
+org.hibernate.validator.constraints.URL.message                     = mus\u00ed b\u00fdt platnou adresou URL
+
+org.hibernate.validator.constraints.br.CNPJ.message                 = neplatn\u00e9 brazilsk\u00e9 registra\u010dn\u00ed \u010d\u00edslo pl\u00e1tce korpor\u00e1tn\u00ed dan\u011b (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = neplatn\u00e9 brazilsk\u00e9 registra\u010dn\u00ed \u010d\u00edslo individu\u00e1ln\u00edho pl\u00e1tce dan\u011b (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = neplatn\u00e9 brazilsk\u00e9 \u010d\u00edslo karty Voter ID
+
+org.hibernate.validator.constraints.pl.REGON.message                = Neplatn\u00e9 polsk\u00e9 identifika\u010dn\u00ed \u010d\u00edslo pl\u00e1tn\u011b dan\u011b (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = Neplatn\u00e9 da\u0148ov\u00e9 identifika\u010dn\u00ed \u010d\u00edslo (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = Neplatn\u00e9 polsk\u00e9 n\u00e1rodn\u00ed identifika\u010dn\u00ed \u010d\u00edslo (PESEL)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: The case is not very consistent here. In the original file, we decided to use capitals for all the parts of the number as it's the translation of an acronym. Here we only have the first element capitalized and not the others. Either we decide to stick to what we did in the English version or we are consistent with lowercase if it's better in the language.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties [hunk 5]
```diff
@@ -1,27 +1,53 @@
...
-javax.validation.constraints.Min.message         = mus\u00ed b\u00fdt v\u011bt\u0161\u00ed ne\u017e nebo rovno {value}
-javax.validation.constraints.NotBlank.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotEmpty.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotNull.message     = nesm\u00ed b\u00fdt null
-javax.validation.constraints.Null.message        = mus\u00ed b\u00fdt null
-javax.validation.constraints.Past.message        = mus\u00ed b\u00fdt v minulosti
-javax.validation.constraints.Pattern.message     = se mus\u00ed shodovat s "{regexp}"
-javax.validation.constraints.Size.message        = velikost mus\u00ed b\u00fdt mezi {min} a {max}
+javax.validation.constraints.AssertFalse.message     = mus\u00ed m\u00edt hodnotu false (ne)
+javax.validation.constraints.AssertTrue.message      = mus\u00ed m\u00edt hodnotu true (ano)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties]: Same as Gunnar, I think we should stick to translating true and false. The message is clear enough IMHO.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 6]
```diff
@@ -1,40 +1,53 @@
...
-javax.validation.constraints.NotEmpty.message       = darf nicht leer sein
-javax.validation.constraints.NotNull.message        = darf nicht null sein
-javax.validation.constraints.Null.message           = muss null sein
-javax.validation.constraints.Past.message           = muss in der Vergangenheit liegen
-javax.validation.constraints.PastOrPresent.message  = muss in der Gegenwart oder Vergangenheit liegen
-javax.validation.constraints.Pattern.message        = muss auf Ausdruck "{regexp}" passen
-javax.validation.constraints.Positive.message       = muss gr\u00F6\u00DFer 0 sein
-javax.validation.constraints.PositiveOrZero.message = muss gr\u00F6\u00DFer oder gleich 0 sein
-javax.validation.constraints.Size.message           = muss zwischen {min} und {max} liegen
+javax.validation.constraints.AssertFalse.message     = muss false sein.
```
[the remark this answers is not in the document]
Comment (gsmet) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: Same here. And valid for all the contributed translations.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties (hunk 3 above)
Comment (gsmet) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: +1

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 7]
```diff
@@ -1,40 +1,53 @@
...
+org.hibernate.validator.constraints.Email.message                   = keine korrekt formatierte E-Mail-Adresse.
+org.hibernate.validator.constraints.ISBN.message                    = Ung\u00fcltiger ISBN-Wert
+org.hibernate.validator.constraints.Length.message                  = L\u00e4nge muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.CodePointLength.message         = L\u00e4nge muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.LuhnCheck.message               = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Luhn Modulo 10-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.Mod10Check.message              = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Modulo 10-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.Mod11Check.message              = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Modulo 11-Kontrollsumme ist fehlgeschlagen.
+org.hibernate.validator.constraints.ModCheck.message                = Die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, ${modType}-Kontrollsumme ist fehlgeschlagen.
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: The case is not consistent with the other messages. I think it should be "die".

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 8]
```diff
@@ -1,40 +1,53 @@
...
+org.hibernate.validator.constraints.NotBlank.message                = darf nicht leer sein.
+org.hibernate.validator.constraints.NotEmpty.message                = darf nicht leer sein.
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = Die Evaluierung des Scriptausdrucks "{script}" ergab nicht true.
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: Same remark for "die" here.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 9]
```diff
@@ -1,40 +1,53 @@
...
+org.hibernate.validator.constraints.Range.message                   = muss zwischen {min} und {max} sein.
+org.hibernate.validator.constraints.SafeHtml.message                = hat m\u00f6glicherweise keinen gesch\u00fctzten HTML-Inhalt.
+org.hibernate.validator.constraints.ScriptAssert.message            = Die Evaluierung des Scriptausdrucks "{script}" ergab nicht true.
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: Same remark for "die" here.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 10]
```diff
@@ -1,40 +1,53 @@
...
+org.hibernate.validator.constraints.br.CNPJ.message                 = Ung\u00fcltige brasilianische Registriernummer f\u00fcr K\u00f6rperschaftssteuerzahlungen (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = Ung\u00fcltige brasilianische Registriernummer f\u00fcr Steuerzahlungen nat\u00fcrlicher Personen (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = Ung\u00fcltige brasilianische ID-Karte f\u00fcr Entscheidungsberechtigte
+
+org.hibernate.validator.constraints.pl.REGON.message                = Ung\u00fcltige polnische Steuerzahleridentifikationsnummer (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = Ung\u00fcltige Mehrwertsteueridentifikationsnummer (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = Ung\u00fcltige polnische nationale Identifikationsnummer (PESEL)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: I believe `Ung\u00fcltige` is not a noun so it shouldn't be capitalized if we want to be consistent.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_es.properties [hunk 11]
```diff
@@ -1,53 +1,53 @@
...
+org.hibernate.validator.constraints.ISBN.message                    = ISBN no v\u00e1lido
+org.hibernate.validator.constraints.Length.message                  = la longitud debe estar entre {min} y {max}
+org.hibernate.validator.constraints.CodePointLength.message         = la longitud debe estar entre {min} y {max}
+org.hibernate.validator.constraints.LuhnCheck.message               = El d\u00edgito de comprobaci\u00f3n para ${validatedValue} no es v\u00e1lido, ha fallado la suma de comprobaci\u00f3n de Luhn Modulo 10
+org.hibernate.validator.constraints.Mod10Check.message              = El d\u00edgito de comprobaci\u00f3n para ${validatedValue} no es v\u00e1lido, ha fallado la suma de comprobaci\u00f3n de Modulo 10
+org.hibernate.validator.constraints.Mod11Check.message              = El d\u00edgito de comprobaci\u00f3n para ${validatedValue} no es v\u00e1lido, ha fallado la suma de comprobaci\u00f3n de Modulo 11
+org.hibernate.validator.constraints.ModCheck.message                = El d\u00edgito de comprobaci\u00f3n para ${validatedValue} no es v\u00e1lido, ha fallado la suma de comprobaci\u00f3n ${modType}
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_es.properties]: Case of `El` inconsistent.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_es.properties [hunk 12]
```diff
@@ -1,53 +1,53 @@
...
+org.hibernate.validator.constraints.br.CPF.message                  = n\u00famero de registro de contribuyente individual de Brasil no v\u00e1lido (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = n\u00famero de tarjeta de ID de votante de Brasil no v\u00e1lido
 
-org.hibernate.validator.constraints.pl.REGON.message                = n\u00FAmero de identificaci\u00F3n tributario Polaco (REGON) inv\u00E1lido
-org.hibernate.validator.constraints.pl.NIP.message                  = n\u00FAmero de identificaci\u00F3n VAT (NIP) inv\u00E1lido
-org.hibernate.validator.constraints.pl.PESEL.message                = n\u00FAmero de identificaci\u00F3n nacional Polaco (PESEL) inv\u00E1lido
+org.hibernate.validator.constraints.pl.REGON.message                = N\u00famero de identificaci\u00f3n de contribuyente de Polonia no v\u00e1lido (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = N\u00famero de identificaci\u00f3n de VAT no v\u00e1lido (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = N\u00famero de identificaci\u00f3n nacional de Polonia no v\u00e1lido (PESEL)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_es.properties]: Same remark for the consistency of the acronym. Either stick to capital for each word of the acronym translation or use lowercase.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 13]
```diff
@@ -1,53 +1,53 @@
...
-javax.validation.constraints.NotNull.message         = ne peut pas \u00EAtre nul
-javax.validation.constraints.Null.message            = doit \u00EAtre nul
-javax.validation.constraints.Past.message            = doit \u00EAtre dans le pass\u00E9
-javax.validation.constraints.PastOrPresent.message   = doit \u00EAtre dans le pass\u00E9 ou dans le pr\u00E9sent
-javax.validation.constraints.Pattern.message         = doit respecter "{regexp}"
-javax.validation.constraints.Positive.message        = doit \u00EAtre strictement positif
-javax.validation.constraints.PositiveOrZero.message  = doit \u00EAtre positif ou \u00E9gal \u00E0 0
-javax.validation.constraints.Size.message            = la taille doit \u00EAtre comprise entre {min} et {max}
+javax.validation.constraints.AssertFalse.message     = doit avoir la valeur false
+javax.validation.constraints.AssertTrue.message      = doit avoir la valeur true
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: Same remark here about `false` and `true`. Valid for all the translations.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 14]
```diff
@@ -1,53 +1,53 @@
...
+javax.validation.constraints.Future.message          = doit \u00eatre une date dans le futur
+javax.validation.constraints.FutureOrPresent.message = Doit \u00eatre une date dans le pr\u00e9sent ou le futur
+javax.validation.constraints.Max.message             = doit \u00eatre inf\u00e9rieur ou \u00e9gal \u00e0 {value}
+javax.validation.constraints.Min.message             = doit \u00eatre sup\u00e9rieur ou \u00e9gal \u00e0 {value}
+javax.validation.constraints.Negative.message        = doit \u00eatre inf\u00e9rieur \u00e0 0
+javax.validation.constraints.NegativeOrZero.message  = doit \u00eatre inf\u00e9rieur ou \u00e9gal \u00e0 0
+javax.validation.constraints.NotBlank.message        = ne doit pas \u00eatre blanc
+javax.validation.constraints.NotEmpty.message        = ne doit pas \u00eatre vide
+javax.validation.constraints.NotNull.message         = ne doit pas avoir la valeur NULL
+javax.validation.constraints.Null.message            = doit avoir la valeur NULL
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: I don't think `la valeur NULL` is an improvement. I can fix it myself as it's the French translation.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 15]
```diff
@@ -1,53 +1,53 @@
...
-javax.validation.constraints.PositiveOrZero.message  = doit \u00EAtre positif ou \u00E9gal \u00E0 0
-javax.validation.constraints.Size.message            = la taille doit \u00EAtre comprise entre {min} et {max}
+javax.validation.constraints.AssertFalse.message     = doit avoir la valeur false
+javax.validation.constraints.AssertTrue.message      = doit avoir la valeur true
+javax.validation.constraints.DecimalMax.message      = doit \u00eatre inf\u00e9rieur \u00e0 ${inclusive == true ? ' ou \u00e9gal \u00e0 ' : ''}{value}
+javax.validation.constraints.DecimalMin.message      = doit \u00eatre sup\u00e9rieur \u00e0 ${inclusive == true ? ' ou \u00e9gal \u00e0 ' : ''}{value}
+javax.validation.constraints.Digits.message          = valeur num\u00e9rique hors limites (<{integer} chiffres>.<{fraction} chiffres> attendu)
+javax.validation.constraints.Email.message           = doit \u00eatre une adresse \u00e9lectronique syntaxiquement correcte
+javax.validation.constraints.Future.message          = doit \u00eatre une date dans le futur
+javax.validation.constraints.FutureOrPresent.message = Doit \u00eatre une date dans le pr\u00e9sent ou le futur
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: Case consistency.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 16]
```diff
@@ -1,53 +1,53 @@
...
+org.hibernate.validator.constraints.ISBN.message                    = ISBN non valide
+org.hibernate.validator.constraints.Length.message                  = la longueur doit \u00eatre comprise entre {min} et {max}
+org.hibernate.validator.constraints.CodePointLength.message         = la longueur doit \u00eatre comprise entre {min} et {max}
+org.hibernate.validator.constraints.LuhnCheck.message               = Le chiffre cl\u00e9 pour ${validatedValue} n'est pas valide ; la somme de contr\u00f4le de l'algorithme de Luhn (modulo 10) a \u00e9chou\u00e9
+org.hibernate.validator.constraints.Mod10Check.message              = Le chiffre cl\u00e9 pour ${validatedValue} n'est pas valide ; la somme de contr\u00f4le de l'algorithme modulo 10 a \u00e9chou\u00e9
+org.hibernate.validator.constraints.Mod11Check.message              = Le chiffre cl\u00e9 pour ${validatedValue} n'est pas valide ; la somme de contr\u00f4le de l'algorithme modulo 11 a \u00e9chou\u00e9
+org.hibernate.validator.constraints.ModCheck.message                = Le chiffre cl\u00e9 pour ${validatedValue} n'est pas valide ; la somme de contr\u00f4le de l'algorithme ${modType} a \u00e9chou\u00e9
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: Case consistency.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 17]
```diff
@@ -1,53 +1,53 @@
...
+org.hibernate.validator.constraints.NotBlank.message                = ne peut pas \u00eatre vide
+org.hibernate.validator.constraints.NotEmpty.message                = ne peut pas \u00eatre vide
 org.hibernate.validator.constraints.ParametersScriptAssert.message  = le script "{script}" n'a pas \u00E9t\u00E9 \u00E9valu\u00E9 \u00E0 vrai
-org.hibernate.validator.constraints.Range.message            = doit \u00EAtre entre {min} et {max}
-org.hibernate.validator.constraints.SafeHtml.message         = peut contenir du HTML dangereux
-org.hibernate.validator.constraints.ScriptAssert.message     = le script "{script}" n'a pas \u00E9t\u00E9 \u00E9valu\u00E9 \u00E0 vrai
-org.hibernate.validator.constraints.UniqueElements.message   = ne doit contenir que des \u00E9l\u00E9ments uniques
-org.hibernate.validator.constraints.URL.message              = URL mal form\u00E9e
+org.hibernate.validator.constraints.Range.message                   = doit \u00eatre compris(e) entre {min} et {max}
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: Let's not add the `(e)`: we don't do it for the other sentences.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties [hunk 18]
```diff
@@ -1,53 +1,53 @@
...
+org.hibernate.validator.constraints.br.CNPJ.message                 = num\u00e9ro de registre d'entreprise contribuable br\u00e9silien non valide (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = num\u00e9ro de registre de contribuable individuel br\u00e9silien non valide (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = num\u00e9ro de carte d'\u00e9lecteur br\u00e9silien non valide
 
-org.hibernate.validator.constraints.pl.REGON.message = num\u00E9ro d'imposition polonais (REGON) invalide
-org.hibernate.validator.constraints.pl.NIP.message = num\u00E9ro d'identification de TVA polonais (NIP) invalide
-org.hibernate.validator.constraints.pl.PESEL.message = num\u00E9ro d'identification national polonais (PESEL) invalide
+org.hibernate.validator.constraints.pl.REGON.message                = Num\u00e9ro d'identification de contribuable polonais non valide (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = Num\u00e9ro de TVA non valide (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = Num\u00e9ro de carte d'identit\u00e9 nationale polonais non valide (PESEL)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties]: Same remark about the case.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_hu.properties [hunk 19]
```diff
@@ -1,27 +1,53 @@
...
+org.hibernate.validator.constraints.Length.message                  = a hossznak a(z) {min} \u00e9s {max} \u00e9rt\u00e9kek k\u00f6z\u00f6tt kell lennie
+org.hibernate.validator.constraints.CodePointLength.message         = a hossznak a(z) {min} \u00e9s {max} \u00e9rt\u00e9kek k\u00f6z\u00f6tt kell lennie
+org.hibernate.validator.constraints.LuhnCheck.message               = A(z) ${validatedValue} ellen\u0151rz\u0151 sz\u00e1mjegye \u00e9rv\u00e9nytelen, a Luhn Modulo 10 ellen\u0151rz\u0151\u00f6sszeg meghi\u00fasult
+org.hibernate.validator.constraints.Mod10Check.message              = A(z) ${validatedValue} ellen\u0151rz\u0151 sz\u00e1mjegye \u00e9rv\u00e9nytelen, a Modulo 10 ellen\u0151rz\u0151\u00f6sszeg meghi\u00fasult
+org.hibernate.validator.constraints.Mod11Check.message              = A(z) ${validatedValue} ellen\u0151rz\u0151 sz\u00e1mjegye \u00e9rv\u00e9nytelen, a Modulo 11 ellen\u0151rz\u0151\u00f6sszeg meghi\u00fasult
+org.hibernate.validator.constraints.ModCheck.message                = A(z) ${validatedValue} ellen\u0151rz\u0151 sz\u00e1mjegye \u00e9rv\u00e9nytelen, a(z) ${modType} ellen\u0151rz\u0151\u00f6sszeg meghi\u00fasult
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_hu.properties]: pretty sure there's a case consistency issue but I don't speak the language :).

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_sk.properties [hunk 20]
```diff
@@ -20,10 +20,10 @@ org.hibernate.validator.constraints.EAN.message                     = nespr\u00e
...
-org.hibernate.validator.constraints.Mod11Check.message              = kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det Modulo 11 zlyhal
-org.hibernate.validator.constraints.ModCheck.message                = kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det ${modType} zlyhal
+org.hibernate.validator.constraints.LuhnCheck.message               = Kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det Luhn Modulo 10 zlyhal
+org.hibernate.validator.constraints.Mod10Check.message              = Kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det Modulo 10 zlyhal
+org.hibernate.validator.constraints.Mod11Check.message              = Kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det Modulo 11 zlyhal
+org.hibernate.validator.constraints.ModCheck.message                = Kontroln\u00e1 \u010d\u00edslica pre ${validatedValue} nie je spr\u00e1vna, kontroln\u00fd s\u00fa\u010det ${modType} zlyhal
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_sk.properties]: Case consistency issue.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties (hunk 7 above)
Comment (gunnarmorling) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]: It's related to the added "." This makes it a regular sentence, in which case the first word should start with an upper-case latter. But as said I'd not add the dot, in which case the "die" should be lower-cased.

**Comment by aguibert** [comment]
hi @gsmet I've pushed up changes to address your review comments. Please have another look when you get time

**Comment by gsmet** [comment]
Rebased, fixed a few additional issues, fixed the tests.

We might not agree on every changes but I tried to be consistent.

@gunnarmorling could you take a look at the updated German translation? From my vague German knowledge, it looks like an improvement. Better check the changes with a diff in your IDE as ignoring whitespace on GitHub does not work very well.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 21]
```diff
@@ -1,40 +1,53 @@
...
-org.hibernate.validator.constraints.Range.message            = muss zwischen {min} und {max} liegen
-org.hibernate.validator.constraints.SafeHtml.message         = k\u00F6nnte unsicheren HTML-Inhalt haben
-org.hibernate.validator.constraints.ScriptAssert.message     = Skriptausdruck "{script}" gab nicht true zur\u00FCck
-org.hibernate.validator.constraints.UniqueElements.message   = darf keine Duplikate enthalten
-org.hibernate.validator.constraints.URL.message              = muss eine g\u00FCltige URL sein
+org.hibernate.validator.constraints.CreditCardNumber.message        = ung\u00fcltige Kreditkartennummer
+org.hibernate.validator.constraints.Currency.message                = ung\u00fcltige W\u00e4hrung (muss eine der folgenden sein: {value})
+org.hibernate.validator.constraints.EAN.message                     = ung\u00fcltiger {type}-Barcode
+org.hibernate.validator.constraints.Email.message                   = muss eine korrekt formatierte E-Mail-Adresse sein
+org.hibernate.validator.constraints.ISBN.message                    = Ung\u00fcltiger ISBN-Wert
```
Comment (gunnarmorling) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]:
```suggestion
org.hibernate.validator.constraints.ISBN.message                    = ung\u00fcltige ISBN
```

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties [hunk 22]
```diff
@@ -1,40 +1,53 @@
...
+org.hibernate.validator.constraints.Mod11Check.message              = die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, Modulo 11-Kontrollsumme ist fehlgeschlagen
+org.hibernate.validator.constraints.ModCheck.message                = die Pr\u00fcfziffer f\u00fcr ${validatedValue} ist ung\u00fcltig, ${modType}-Kontrollsumme ist fehlgeschlagen
+org.hibernate.validator.constraints.NotBlank.message                = darf nicht leer sein
+org.hibernate.validator.constraints.NotEmpty.message                = darf nicht leer sein
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = die Evaluierung des Scriptausdrucks "{script}" ergab nicht true
+org.hibernate.validator.constraints.Range.message                   = muss zwischen {min} und {max} sein
+org.hibernate.validator.constraints.SafeHtml.message                = hat m\u00f6glicherweise keinen gesch\u00fctzten HTML-Inhalt
+org.hibernate.validator.constraints.ScriptAssert.message            = die Evaluierung des Scriptausdrucks "{script}" ergab nicht true
+org.hibernate.validator.constraints.UniqueElements.message          = dar nur eindeutige Elemente enthalten
```
Comment (gunnarmorling) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_de.properties]:
```suggestion
org.hibernate.validator.constraints.UniqueElements.message          = darf nur eindeutige Elemente enthalten
```

**Comment by aguibert** [comment]
@gsmet pushed up both suggested edits to the German file.  Ready for re-review now

**Comment by gsmet** [comment]
Merged a slightly adjusted version.

Thanks.

**Pull request closed without merging**